app/controllers/follower_followed_controller.rb

```diff
@@ -0,0 +1,16 @@
+class FollowerFollowedController < ApplicationController
+  before_action :require_current_user
+
+  def create
+    FollowerFollowed.create!(follower_id: current_user.id, followed_id: params[:id])
+    flash[:notice] = "You have followed #{User.where(id: params[:id])[0].name}!"
+    redirect_to user_dashboard_path
+  end
+
+  def destroy
+    FollowerFollowed.where(follower_id: current_user.id, followed_id: params[:id])[0].destroy
+    flash[:notice] = "You have unfollowed #{User.where(id: params[:id])[0].name}!"
+    redirect_to user_dashboard_path
+  end
```

Comment:
This controller handles updating the FollowerFollowed joins table ... which needs a better name in the future.